plugins/channel_header_call/channel_header_call.jsx

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/* eslint-disable react/no-multi-comp */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Dropdown, Tooltip} from 'react-bootstrap';
+import {RootCloseWrapper} from 'react-overlays';
+import {FormattedMessage} from 'react-intl';
+
+import HeaderIconWrapper from 'components/channel_header/components/header_icon_wrapper';
+import PluginChannelHeaderIcon from '../../components/widgets/icons/plugin_channel_header_icon';
+import {Constants} from 'utils/constants';
+import OverlayTrigger from 'components/overlay_trigger';
+
+class CustomMenu extends React.PureComponent {
```

Comment:
Don't we already have a menu for the existing plugin button? Can this share code this that?

Also, to give you a heads up, Apps team is going to be working on writing a reusable menu, so hopefully we can avoid having every component implement its own menu in the future.